app/models/invoice.rb

```diff
@@ -22,4 +22,8 @@ def total_revenue
   def total_revenue_to_currency
     ActiveSupport::NumberHelper::number_to_currency(total_revenue.to_f / 100)
   end
+
+  def self.status_list_for_select_menu
+    statuses.keys.map { |status| [status.titleize, status]}
```

Comment:
Is this Ruby? / Are we allowed to use Ruby on this project?

Reply:
we got the okay from chris on this and I added the commit from chris